src/styles/getMuiTheme.js

```diff
@@ -224,4 +226,12 @@ export default function getMuiTheme(baseTheme, muiTheme) {
       borderColor: palette.borderColor,
     },
   }, muiTheme);
+
+  if (computedTheme.isRtl) {
```

Comment:
> I think this will be very limiting. Some logics related to rtl is simply not implementable with style modification. 

I completely agree, `isRtl` is affecting the style but also the component order and which component we use.
I think it's fine like this.